src/scenarios/shared/precommands.py

```diff
@@ -288,6 +288,37 @@ def uninstall_workload(self, workloadid: str):
         if self.has_workload and not self.readonly_dotnet:
             subprocess.run(["dotnet", "workload", "uninstall", workloadid])
 
+    def create_nuget_config(self, feeds: dict[str, str]={"nuget.org": "https://api.nuget.org/v3/index.json"}):
```

Comment:
We will likely get workitems to pop up if we use nuget.org, would it instead be possible to instead use the one at the top level of the performance repo? If there are some packages that only occasionally update but they are not being found in our feeds, we can have them added.

Also, this does not appear to currently be called anywhere.

Reply:
removed